cypress.config.js

```diff
@@ -11,8 +13,13 @@ module.exports = defineConfig({
       }
     },
 
+    downloadsFolder: "cypress/downloads",
```

Comment:
Explicit mention of the downloads folder location where test will put the downloaded json file from the download page.